src/utils/errorReporting.ts

```diff
@@ -35,16 +36,99 @@ const reportError = (
   };
 };
 
+type Metadata = {
+  [key: string]: any;
+};
+
 const leaveBreadcrumb = (
   message: string,
-  metadata: { [key: string]: any },
+  metadata: Metadata,
   type: BreadcrumbType
 ) => {
   if (!isProductionBuild()) {
     console.debug({ message, metadata, type });
   } else {
     Bugsnag.leaveBreadcrumb(message, metadata, type);
+    addBreadcrumb(convertToSentryBreadcrumb(message, metadata, type));
```

Comment:
does the timestamp field get auto-populated? :0 (asking bc `Bugsnag.leaveBreadcrumb` doesn't have a timestamp field but Sentry does so I was wondering if that means anything)

Reply:
I think without a `timestamp` field the time is populated with the time the error is sent? It's not very clear... but I'll add the field!

Reply:
either one is fine with me as long as its there!! =)